src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3056,26 +3050,16 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return flags;
     }
 
-    til::point TermControl::_toControlOrigin(const til::point terminalPos)
+    winrt::Windows::Foundation::Point TermControl::_toControlOrigin(const til::point terminalPos)
     {
         const auto fontSize{ CharacterDimensions() };
-
-        // Convert text buffer cursor position to client coordinate position
-        // within the window. This point is in _pixels_

```

Comment:
I don't understand why this is no longer true

Reply:
If you meant why I removed "This point is in _pixels_": The returned value is now in DIPs instead of pixels.